msal/application.py

```diff
@@ -413,6 +413,7 @@ def acquire_token_silent(
         """
```

Comment:
We will also need to come up with parameter documentation. Something along these lines:

```
        :param error_response:
            A boolean flag for whether returning error response. 
            Default value is False, which will just return None instead of error.

        :return:
            - A dict containing "access_token" key, when cache lookup succeeds.
            - None when cache lookup does not yield anything.
            - A dict containing the error response,
               this will be available only when an error actually happens,
               and when the "error_response" parameter was True.
```